src/components/BookActions.js

```diff
@@ -0,0 +1,26 @@
+// BookActions.js
```

Comment:
- [ ] It would be nice if you made your pull request summary detailed to make your pull request more descriptive and professional 😊